account_reconciliation_widget/models/reconciliation_widget.py

```diff
@@ -340,18 +334,17 @@ def get_bank_statement_data(self, bank_statement_line_ids, srch_domain=None):
             [line["st_line"]["id"] for line in results["lines"]]
         )
         bank_statements_left = bank_statement_lines_left.mapped("statement_id")
-
+        data = bank_statements_left.read(["name", "journal_id"])
+        data = data and data[0]
```

Comment:
In case of empty list it could be better to fallback to an empty dict so we can use `get` later.
```suggestion
        data = bank_statements_left.read(["name", "journal_id"]) or {}
```